core-metadata/src/main/java/org/apache/kylin/metadata/MetadataConstants.java

```diff
@@ -52,5 +52,6 @@
     public static final String TABLE_EXD_DELIM = "delim";
     public static final String TABLE_EXD_DEFAULT_VALUE = "unknown";
 
+    @Deprecated
```

Comment:
Just remove this constant?

Reply:
Ok. I was worried about other new code using this. I will remove this.